standard/classes.md

```diff
@@ -2777,6 +2779,27 @@ A compile-time error occurs unless all of the following are true for an override
 > ```
 >
 > *end example*
+<!-- markdownlint-disable MD028 -->
+
+<!-- markdownlint-enable MD028 -->
+> *Example*: The following demonstrates how the overriding rules work when type parameters are involved:
+>
+> <!-- Example: {template:"standalone-lib-without-using", name:"OverrideMethods5"} -->
+> ```csharp
+> #nullable enable
+> class A
+> {
+>     public virtual void Foo<T>(T? value) where T : class { }
+>     public virtual void Foo<T>(T? value) where T : struct { }
+> }
+> class B: A
+> {
+>     public override void Foo<T>(T? value) where T : class { }
+>     public override void Foo<T>(T? value) where T : struct { }
+> }
+> ```
+>
+> Without the type parameter constraints in the overriding methods, the compiler won’t know which base method is being overridden. *end example*
```

Comment:
I feel that these two explanatory lines (one here, one in the interfaces example) are misleading. 

It's not about not knowing which base method is being overridden; it's about `T?` being required to mean `System.Nullable<T>` or else breaking valid C# 7 code. `T?` must always mean `System.Nullable<T>` by default, and thus if you want it to mean "nullable reference type" you must use `where T : class`. (Or in C# 9, `where T : default` to mean "defaultable unconstrained type parameter.")

See <https://github.com/dotnet/csharpstandard/pull/671#discussion_r2151047975> for a more in-depth comment.